net/radius-mac/files/radius-mac.upgrade

```diff
@@ -0,0 +1 @@
+/etc/config/radius-mac
```

Comment:
This looks like leftover for me.

Reply:
🤦 yup.  I forgot to add the `git rm` to clean that up.